materialize-iceberg/.snapshots/TestValidateAndApplyLowercaseColumnNames

```diff
@@ -0,0 +1,362 @@
+Big Schema Initial Constraints:
```

Comment:
This snapshot is exactly the same as the `TestValidateAndApply` one, except in the table schema outputs (note: not the constraint outputs, since those are based on Flow field names) the column names are lowercase - which is the expected outcome.